tests/conftest.py

```diff
@@ -53,6 +53,48 @@ def pytest_logger_config(logger_config):
     logger_config.set_formatter_class(OCSLogFormatter)
 
 
+@pytest.fixture()
+def supported_configuration():
+    """
+    Check that cluster nodes have enough CPU and Memory as described in:
+    https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.2/html-single/planning_your_deployment/index#infrastructure-requirements_rhocs
+    This fixture is intended as a prerequisite for tests or fixtures that
+    run flaky on configurations that don't meet minimal requirements.
+
+    Minimum requirements for each starting node (OSD+MON):
+        16 CPUs
+        64 GB memory
+    Last documentation check: 2020-02-21
+    """
+    min_cpu = 16
+    min_memory = 64 * 10**9
+
+    node_obj = ocp.OCP(kind=constants.NODE)
+    log.debug('Checking if system meets minimal requirements')
+    nodes = node_obj.get().get('items')
+    for node_info in nodes:
+        real_cpu = int(node_info.get('status').get('capacity').get('cpu'))
+        real_memory = node_info.get('status').get('capacity').get('memory')
```

Comment:
Using `.get().get()` chains like this works, but in case it fails could only lead to misleading errors when the field is not present (None has not ...). Compared to that, unhandled `[key][key]` will at least provide information about the missing key.

Reply:
done